src/ui/public/kbn_top_nav/kbn_top_nav_controller.js

```diff
@@ -39,16 +45,16 @@ export default function ($compile) {
     }
 
     // little usability helpers
-    which() { return this.currentKey; }
-    is(key) { return this.which() === key; }
-    open(key) { this.set(key); }
-    close(key) { (!key || this.is(key)) && this.set(null); }
-    toggle(key) { this.set(this.is(key) ? null : key); }
+    getCurrent() { return this.currentKey; }
+    isCurrent(key) { return this.getCurrent() === key; }
+    open(key) { this.setCurrent(key); }
+    close(key) { (!key || this.isCurrent(key)) && this.setCurrent(null); }
+    toggle(key) { this.setCurrent(this.isCurrent(key) ? null : key); }
 
     // apply the defaults to individual options
     _applyOptDefault(opt = {}) {
       return defaults({}, opt, {
-        label: capitalize(opt.key),
+        label: opt.label || capitalize(opt.key),
```

Comment:
I should test this, but this shouldn't be necessary, if you set `opt.label` on the raw object due to `_.defaults`. [docs](https://lodash.com/docs#defaults)